### PR TITLE
document platform 8 screenshot automation guidelines

### PR DESCRIPTION
**Pull request by christinaausley**
## What is the purpose of the change

Documenting current screenshot automation efforts in `documentation-guidelines.md`.

This includes efforts across [Modeler](https://github.com/camunda/camunda-docs-modeler-screenshots/blob/main/README.md), Zeebe (according to the approved and [executed](https://github.com/camunda/camunda-platform-docs/pull/1566) [standardization proposal](https://docs.google.com/document/d/1gATpZXVxGiWEF4HfUbpUpQJ8ElyeZROERwiArPX5FRk/edit?usp=sharing)), [Operate and Tasklist](https://github.com/camunda/camunda-docs-operate-tasklist-screenshots), and [Optimize](https://github.com/camunda/camunda-optimize).

**My question: Would this be better suited in the contribution guidelines?**

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by akeller — COMMENTED** [review]
This looks great! Thank you for adding some forward looking comments on the other components as well. 

For future components is there a generic process they should follow? Should they follow the Zeebe process?

Could you add a couple of sentences about how (or if) individuals should contribute screenshot updates or new screenshots via PR? I'm thinking of a scenario where a community member wants to update a screenshot manually. Do we accept it? Or do we take it as a sign we need to run the screenshot automation process for the relevant component?

**Comment by akeller** [comment]
> My question: Would this be better suited in the contribution guidelines?

I missed this before and I think you could make an argument either way. Not sure I have strong enough feelings to want to suggest moving this.

**Review by akeller — DISMISSED** [review]

**Comment by akeller** [comment]
Ready to merge this @christinaausley?

File: howtos/documentation-guidelines.md [hunk 1]
```diff
@@ -162,6 +162,43 @@ If you wish to test `.htaccess` rules, you have a couple options:
 
    4. Clean up the server with `docker compose down`.
 
+## Screenshot automation
+
+In an effort to automate screenshots across Camunda Platform 8 documentation, the following teams execute uniform steps when incorporating images and diagrams:
+
+:::note
```
Comment (christinaausley) [inline howtos/documentation-guidelines.md:169]: @akeller Would you word this differently?

Comment (akeller) [reply howtos/documentation-guidelines.md:169]: I'm good with this! We can always iterate 😄

**Review by akeller — APPROVED** [review]

**Pull request merged**